mw-webapp/src/component/confirm/Confirm.cy.tsx

```diff
@@ -0,0 +1,104 @@
+import {Button} from "src/component/button/Button";
+import {Confirm} from "src/component/confirm/Confirm";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+
+const CONFIRM_CY = {
+  dataCyTrigger: "trigger",
+  dataCyContent: {
+    dataCyOverlay: "overlay",
+    dataCyClose: "close",
+    dataCyContent: "content",
+    dataCyOK: "confirm",
+  },
+  btnCancel: "cancel",
+  btnOk: "confirm",
+};
+
+describe("Confirm component", () => {
+
+  beforeEach(() => {
+    cy.mount(
+      <Confirm
+        cy={CONFIRM_CY}
+        trigger={
+          <Button
+            value="Confirm trigger"
+            onClick={() => {}}
+          />
+        }
+        content={
+          <div>
+            Confirm content
+          </div>
+        }
+        onOk={() => {}}
+        okText="Test confirm"
+      />
+      ,
+    );
+  });
+
+  it("should not exist by default", () => {
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyContent))
```

Comment:
lets use helper "getDataCy" for getting elements by cy attribute

Reply:
Didn't understand. I guess that's what I do